### PR TITLE
Close #138: use "tag content" preferentially.

### LINKED ISSUE #138: "enclosed" vs "tag content"
The terminology used varies a little from "enclosed value" to "enclosed data item" to "tag content". I think "tag content" is now the preference. Search for "enclosed" to find these.

### PR DESCRIPTION
**Pull request by cabo**
Apparently, I forgot to create the actual PR.

### CODE REVIEW

**Comment by paulehoffman** [comment]
Hopefully improved some of the new wording

File: draft-ietf-cbor-7049bis.md [hunk 1]
```diff
@@ -904,7 +904,7 @@ Tag number 1 contains a numerical value counting the number of seconds
 from 1970-01-01T00:00Z in UTC time to the represented point in civil
 time.
 
-The tag content data item MUST be an unsigned or negative integer (major types 0
```
Comment (cabo) [inline draft-ietf-cbor-7049bis.md]: I occasionally used “tag content data item” for emphasis that the tag content is a data item.  Too much redundancy?

Comment (paulehoffman) [reply draft-ietf-cbor-7049bis.md]: Yes, definitely.

File: draft-ietf-cbor-7049bis.md [hunk 2]
```diff
@@ -1152,7 +1152,7 @@ help to have some distinguishing characteristics for the data itself.
 
 Tag number 55799 is defined for this purpose.  It does not impart any special
 semantics on the data item that it encloses; that is, the semantics of
-the tag content data item enclosed in tag number 55799 is exactly identical to the semantics
+the tag content enclosed in tag number 55799 is exactly identical to the semantics
 of the data item itself.
```
Comment (cabo) [inline draft-ietf-cbor-7049bis.md]: But not here!

Comment (paulehoffman) [reply draft-ietf-cbor-7049bis.md]: We disagree, but if you really want to revert, go ahead.

**Pull request merged**